README.md

```diff
@@ -1,2 +1,46 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+## The aim of this pseudocode is to show the process of selecting a secret word from a string of other words. Which has been selected by another user. 
```

Comment:
The worksheet asked for pseudocode for the algorithm to check the likeness score, not for the whole game! Careful reading of specifications is an important skill for software developers to master.